contents/blog/the-posthog-array-1-30-0.md

```diff
@@ -0,0 +1,130 @@
+---
+date: 2021-11-17
+title: Array 1.30.0
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Release notes"]
+featuredImage: ../images/blog/array/1-28-0.png
+excerpt: Excerpt here
+---
+
+
+PostHog 1.30.0 includes a description here.
+
+## PostHog 1.30.0 release notes
+
+> For a better experience and new features, remember to [update your PostHog instance](/docs/self-host/configure/upgrading-posthog).
+
+**Release highlights:**
+- [Fresh new look-and-feel](#fresh-new-look-and-feel)
+- [Correlation analysis](#correlation-analysis)
+- [Saved insights](#saved-insights)
+- [Fully revamped recordings](#fully-revamped-recordings)
+
+### Fresh new look-and-feel
+
+PostHog just got a lot fresher! We have a brand new UI and layout that had been on the works, codename `lemonade` (because it's fresh). It's not only a coat of paint, we've pushed a lot of improvements to overall navigation and performance, we call it turbo mode.
+
+
+{images here}
+
+<br />
+
+### Correlation analysis
+
+Trying to understand why your users are converting or dropping? Presenting Correlation analysis. This nifty new insight will automatically match your funnels to any relevant signals on why your users convert or drop (e.g. "Users in Canada are 5x more likely to convert" or "Users in Chrome are 3x less likely to convert"). Funnel optimization to the next level. [Read more here](/docs/....).
```

Comment:
@neilkakkar would be awesome if we had some docs for the release on Wed

Reply:
Neil is out this week. @hazzadous can help perhaps?

Reply:
Yes can add something. Do you have logs that you think our great that I can use as a template @paolodamico ?

Reply:
@paolodamico will something like this do? https://github.com/PostHog/posthog.com/pull/2445/files

Reply:
This is great @hazzadous! Added some comments and will link to the new place in the docs already